components/map/map.js

```diff
@@ -64,7 +64,7 @@ export default class Map extends React.Component {
 
     // Add layers
     this.initLayerManager();
-    this.props.layers && this.addLayer(this.props.layers);
+    this.props.layers && this.props.layers.length && this.addLayer(this.props.layers);
```

Comment:
You can avoid having this conditions by declaring inside the default props the layers as an array